tdrs-frontend/src/components/FileUpload/FileUpload.jsx

```diff
@@ -13,7 +13,7 @@ import createFileInputErrorState from '../../utils/createFileInputErrorState'
 const INVALID_FILE_ERROR =
   'We can’t process that file format. Please provide a plain text file.'
 
-function FileUpload({ section, setlocalAlertState }) {
+function FileUpload({ section, setLocalAlertState }) {
```

Comment:
Note I'm just renaming these all to fix a typo `setlocalAlertState` => `setLocalAlertState`